src/header/header.scss

```diff
@@ -25,159 +25,105 @@
     color: t(iui-text-color);
   }
 
-  > .iui-left,
-  > .iui-center,
-  > .iui-right {
-    display: flex;
-    align-items: center;
-    height: 100%;
-  }
-
-  @include iui-header-buttons;
-
-  > .iui-left {
-    > nav {
-      display: flex;
-      align-items: center;
-      height: 100%;
-    }
-
-    &,
-    > nav {
-      overflow: hidden;
-      box-sizing: content-box;
-      margin-bottom: -$iui-xs;
-      padding-bottom: $iui-xs;
-    }
+  &.iui-slim {
+    height: round($iui-baseline * 2.5) + $iui-xxs; // Remove `+ $iui-xxs` when box-sizing: border-box is applied to all elements.
```

Comment:
Mybe we could apply that bordex-box where needed and remove these 'hacks'?

Reply:
When I started this PR I actually removed that hack, but it caused test images to change so I left it as is.  I think our original plan was when we add `* {box-sizing: border-box;}` we'd remove the hacky CSS because multiple test images would be changing.

Reply:
I kinda like that no images changed in this PR
![image](https://user-images.githubusercontent.com/9084735/154554314-db5ea792-a08b-4fac-8763-92654b1d34c4.png)